samples/wifi/shutdown/README.rst

```diff
@@ -28,6 +28,15 @@ The sample:
 #. Shuts down the Wi-Fi driver.
 #. Puts the nRF5340 SoC in the lowest possible power consumption mode.
 
+Shutdown mode
+*************
+
+The Wi-Fi chipset power state is automatically managed by the Wi-Fi driver, if the network interface is down the Wi-Fi chipset is put in `shutdown` mode.
+And if the network interface is up the Wi-Fi chipset is put in `active` mode.
+
+The `shutdown` here refers to de-asserting `buck_en` signal to the Wi-Fi chipset powering the Wi-Fi chipset down.
```

Comment:
Let's make this a bit more verbose:
"As part of transition to `Shutdown` state, the Wi-Fi driver de-asserts the `buck_en` signal to the nRF70 Series chipset to completely power down the device. When requested to power on the device, the Wi-Fi driver will re-assert the `buck_en` signal to the nRF70 Series chipset."

Reply:
And I would also add a note, of the impact of power down, when it comes to the ram retention.

Proposal for a "note" @krish2718 @richabp 
"In Shutdown state the nRF70 Series device does not retain any state information. Powering up the device after Shutdown invokes the complete nRF70 Series boot up sequence."

Reply:
Isn't this covered in PS?

Reply:
It is actually covered even in the NCS docs (Operating on power save). But it does not harm adding a note here. MY point is to make clear that as part of the transition from Shutdown to Active (via net if up, or whatever...) the Wi-Fi driver will perform the proper boot up. This is not covered in PS, as this is mostly SW-related info. We may have it on the "Working with nRF70 Series", but a repetition in this sample does not harm.

Reply:
I don't want to repeat details instead re-use, so, buck-en stuff is covered in PS and I doubt users will understand that without additional context which again is available in the PS.

Reply:
Based on your last comment: how about removing also the above sentence and putting this in the Power States section, in the "Operating in Power Save Modes" chapter in "Working with nRF70 Series"?

Reply:
That was the main intention of the PR as the source code doesn't have any `poweroff` API, so, just a line to tie power with iface.